sirepo/pkcli/admin.py

```diff
@@ -77,44 +77,60 @@ def move_user_sims(target_uid=''):
         shutil.move(lib_file, target)
 
 
-def purge_guest_users(days=180, confirm=False):
-    """Remove old users from db which have not registered.
+def purge_free_users(days=180, confirm=False):
+    """Remove basic users simulations
 
     Args:
         days (int): maximum days of untouched files (old is mtime > days)
         confirm (bool): delete the directories if True (else don't delete) [False]
 
     Returns:
-        (list, list): dirs and uids of removed guest users (or to remove if confirm)
+        (list, list): dirs and uids of removed users (or to remove if confirm)
     """
 
+    def _check_expired_run_dir(path):
+        d = str(path).lower()
+        if 'report' in d or 'animation' in d:
```

Comment:
I don't see why this code needs to concern itself with the simulation run directory name.